packages/php-wasm/logger/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './logger';
```

Comment:
Nitpick: newlines

Reply:
I see now, for some reason, I had auto-linting disabled.

Reply:
[1d840ed](https://github.com/WordPress/wordpress-playground/pull/1035/commits/1d840ed57134daf4c338bf85439a591ab4512477)